src/main/java/com/minecolonies/coremod/client/gui/WindowClipBoard.java

```diff
@@ -119,7 +119,9 @@ public void onOpened()
 
             if (!displayStacks.isEmpty())
             {
-                exampleStackDisplay.setItem(displayStacks.get((lifeCount / LIFE_COUNT_DIVIDER) % displayStacks.size()));
+                if(exampleStackDisplay != null) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)